src/Network.cpp

```diff
@@ -263,15 +263,15 @@ std::pair<int, int> Network::load_network(std::istream& wtfile) {
     while (std::getline(wtfile, line)) {
         std::vector<float> weights;
```

Comment:
Might be a win to reserve more space for weights too.

Reply:
![image](https://user-images.githubusercontent.com/4205182/38762982-140914da-3f61-11e8-93e1-7f727c878a19.png)

My profile shows that it doesn't do too much for me